### ISSUE TITLE
winston detetction not sufficient

### ISSUE THREAD

**Issue by daniel-white** [RobinTail/express-zod-api#1605]
# Description

i'm trying to use pino over winston and I have to hack the logger instance to look not like the simpleified winston config.

```
  // HACK
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  (logger as any).color = undefined;
```

Without this, I get

```
X [MissingPeerError]: Missing peer dependency: winston. Please install it to use the feature.
    at ne (/Users/danielwhite/Repos/prism-service/node_modules/express-zod-api/dist/index.cjs:5:8015)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Mr (/Users/danielwhite/Repos/prism-service/node_modules/express-zod-api/dist/index.cjs:18:1169)
    at async Nr (/Users/danielwhite/Repos/prism-service/node_modules/express-zod-api/dist/index.cjs:18:1838)
```

## Expected

I would not expect express-zod-api to allow pino with no modifications

## Reproduction

```
import pino, { BaseLogger, LevelWithSilent, LoggerOption } from 'pino';

export function createLogger(level: LevelWithSilent): BaseLogger {
  const options: LoggerOptions = {
    level
  };
  const logger = pino(options);
  return logger;
}
```

## Context

- Node.js version: 20.11.1 
- `express-zod-api` version: 17.2.0

**Comment by RobinTail** [comment]
how do you use that function `createLogger` that you made?
Please provide more context for reproducing the issue, @daniel-white 

Please also read the corresponding article in Readme:
https://github.com/RobinTail/express-zod-api?tab=readme-ov-file#customizing-logger

**Comment by RobinTail** [comment]
I think I'm getting now what's happening there.
Please stand by....

**Comment by RobinTail** [comment]
I made a test for the function `isSimplifiedWinstonConfig()` against the pino instance as `pino({level: "info"})`

https://github.com/RobinTail/express-zod-api/pull/1607

And that test passes. The function basically tells whether or not should `winston` be loaded instead.
Can not reproduce that behaviour so far.

What is your actual version of `pino`, @daniel-white 

Could you run `yarn why pino` and tell me what is says?

**Comment by daniel-white** [comment]
hm interesting i'm using 8.19.0 as well. looks like a good test to have and ill keep playing on my end.

**Comment by RobinTail** [comment]
Are you sure you're supplying "info" to `createLogger()`?
Not one of these: `"silent", "warn", "debug"` , @daniel-white

**Comment by RobinTail** [comment]
Since when it's "debug", the test fails and that's I believe exactly the case, @daniel-white

**Comment by daniel-white** [comment]
@RobinTail i was doing a bit more. i'm getting that value from envvars, i opened it up to more levels. it might have been on debug yesterday

I'm allowing
```
['error', 'warn', 'info', 'debug', 'trace']
```

**Comment by RobinTail** [comment]
@daniel-white ,
🚀 the issue fixed in the recently released version 17.2.1 .
Thank you for the contribution!
